modules/troubleshooting/pages/log-files.adoc

```diff
@@ -1,107 +1,111 @@
-= Log files
+= Log Files
 
-TigerGraph Database captures key information on activities occurring across its different components through log functions that output to log files. These log files are not only helpful in xref:troubleshooting-guide.adoc[troubleshooting] but also serve as an auditory resource. This document gives a high-level overview of TigerGraph's logging structure and lists some common information one might need to monitor their database services and where to obtain them in the logs.
+The TigerGraph database captures key information on activities occurring across its different components through log functions that output to log files.
+These log files are not only helpful in xref:troubleshooting-guide.adoc[troubleshooting] but also serve as a resource for auditing.
 
-== Overall Logging Structure
+This page provides a general overview of the way log files are stored in TigerGraph.
 
-Logs in TigerGraph are stored at `<tigergraph_root_dir>/log/`. TigerGraph's logs are divided into different folders by the different internal components and each folder corresponds to a different component. Log formats also vary across the different components. In folders where logs are checked often, such as `restpp`, `gsql`, and `admin`,  there are three symbolic links that help you quickly get to the most recent log file of that category:
+For information and examples specific to audit logging of RESTPP requests, queries, and user management tasks, see xref:audit-logs.adoc[].
 
-* `log.INFO`
- ** Contains regular output and errors
-* `log.ERROR`
- ** Contains errors only
-* `<component_name>.out`
- ** Contains all output from the component process
-* `log.WARNING` or `log.DEBUG`
- ** `log.WARNING` contains warnings
- ** In the``gsql`` folder, `log.DEBUG` contains very specific information you only need when certain errors happen
+== TigerGraph log structure
 
-Knowing where certain activities are recorded allows one to use tools such as the Linux `grep` command to easily obtain critical information from your database.
+Logs in TigerGraph are stored at `<tigergraph root directory>/log/`, which by default is `/home/tigergraph/tigergraph/log`.
```

Comment:
Nit: there is no single root directory for TG, there's temp/data/app/log roots any of which can be on its own partition. So I think this should be rephrased as something like: "the logs are stored in TigerGraph's log root which is configured at install time. You can find the location using gadmin config get System.LogRoot"